core/src/java/org/jdom/input/SAXBuilder.java

```diff
@@ -810,6 +810,14 @@ private void setFeaturesAndProperties(XMLReader parser,
         }
         catch (SAXNotRecognizedException e) { /* Ignore... */ }
         catch (SAXNotSupportedException  e) { /* Ignore... */ }
+
+        // Set any user-specified features on the parser.
+        iter = features.keySet().iterator();
+        while (iter.hasNext()) {
+            String  name  = (String)iter.next();
+            Boolean value = (Boolean)features.get(name);
+            internalSetFeature(parser, name, value.booleanValue(), name);
+        }
```

Comment:
Should also remove the redundant setting of user-specified features from above this

Reply:
Done. I didn't do that initially just to minimise the code change, but that's ok.